pandas/tests/indexes/test_base.py

```diff
@@ -1206,6 +1206,16 @@ def test_is_object(self, index, expected):
     def test_is_all_dates(self, index, expected):
         assert index.is_all_dates is expected
 
+    @pytest.mark.parametrize(
```

Comment:
can you just add these onto the above test

Reply:
Do you want to merge those two tests into one or just to extend the list of fixtures in test_is_all_dates?

Reply:
I extended the test above to check if `is_all_dates==True` for all of the date types. This was indeed missing.
But the test I added checks for something different, what was exposed in the bug report: if `is_all_dates` remains consistent for every date type index during manipulation.